File: C programming/Competitive/Assignments/Assignment_11/Que_2.c
//question 2: Write a program which accepts starting and ending point from user and display all even numbers in between that range.

#include<stdio.h>

void RangeEvenDisplay(int iStart, int iEnd)
{
    if(iStart > iEnd)
    {
        printf("Invalid range\n");
        return;
    }

    for(int i = iStart; i <= iEnd; i++)
    {
        if(i % 2 == 0)
        {
            printf("%d ", i);
        }
    }
    printf("\n");
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter starting point: ");
    scanf("%d",&iValue1);

    printf("Enter ending point: ");
    scanf("%d",&iValue2);

    RangeEvenDisplay(iValue1, iValue2);

    return 0;
}

File: Java Programming/Competitve/Assignments/Asssignment_18/Que_2.java
//Write a program to print all even numbers up to N

class Logic 
{
    void printEvenNumbers(int n) 
    {
        for (int i = 1; i <= n; i++)
         {
            if (i % 2 == 0)
             {
                System.out.print(i + " ");
             }
        }
        System.out.println();
    }
}


class Que_2
{
    public static void main(String A[])
    {
        Logic obj = new Logic();
        obj.printEvenNumbers(20);
    } 
}